cloudfunctions/startGameV2/index.js

```diff
@@ -23,7 +23,10 @@ exports.main = async (event, context) => {
     if (openids.length < 7) { throw Error('房间人数不足，无法开始游戏') }
 
     var sortedOpenids = openids
-    sortedOpenids.sort(_ => { return .5 - Math.random() })
+    for (let i = 1; i < sortedOpenids.length; i++) {
```

Comment:
You should iterate from the last to the first. The algorithm logic is to randomly select one from all unsorted numbers and replace it with the current position.

Reply:
Yes. I changed it locally but forgot to add it when force pushed it.